src/track.rs

```diff
@@ -140,10 +140,14 @@ impl Client {
     async fn docker(&self, id: &str) -> Option<Container> {
         let c = self.docker.as_ref()?.containers();
         let c = c.get(id).inspect().await.ok()?;
+        let ns = "";
+        let pod = "";
         Some(Container {
             id:    c.id,
             name:  c.name,
             image: c.config.image,
+            namespace: ns.to_string(),
+            pod: pod.to_string(),
```

Comment:
The temporary `ns` and `pod` variables don't seem too helpful here, perhaps just do `namespace: "".to_string()`? But see the first comment, they may be better off in a dedicated `Pod` struct that is None when the container is running in Docker not k8s.